cmd/ingress-trafikverket/main.go

```diff
@@ -186,21 +174,18 @@ func main() {
 
 	authKeyEnvironmentVariable := "TFV_API_AUTH_KEY"
 	authenticationKey := os.Getenv(authKeyEnvironmentVariable)
+	trafikverketURL := os.Getenv("TFV_API_URL")
+	contextBrokerURL := os.Getenv("CONTEXT_BROKER_URL")
 
 	if authenticationKey == "" {
 		log.Fatal("API authentication key missing. Please set " + authKeyEnvironmentVariable + " to a valid API key.")
 	}
 
```

Comment:
Please add similar checks for the presence of trafikverket and context broker URL values.